TODO.md

```diff
@@ -0,0 +1,8 @@
+- [x] SDL should extend type for external types - I guess marking types in SDL
+  - [ ] can't generate graphql-js stuff, don't want to do it for externs - don't support graphql-js for this?
+- [ ] all imported types (so support interfaces etc)
+- [ ] Read SDL to actually do validation
+  - [ ] reenable global validations
+- [ ] "modular" mode? like no full schema, but parts of schema but with full validation by resolving it?
+- [ ] all tests to add fixtures for metadata/resolver map
```

Comment:
I'm onboard with this. Happy to have this as a separate PR if you want to merge that first.